lib/omniauth/strategies/openid_connect.rb

```diff
@@ -378,7 +378,7 @@ def verify_id_token!(id_token)
 
         decode_id_token(id_token).verify!(issuer: options.issuer,
                                           client_id: client_options.identifier,
-                                          nonce: stored_nonce)
+                                          nonce: params['nonce'].presence || stored_nonce)
```

Comment:
Isn't presence only a Rails-specific method? I'm not sure how the test case below passed.

```ruby
% irb
irb(main):001:0> foo = "hi"
=> "hi"
irb(main):002:0> foo.presence
(irb):2:in `<main>': undefined method `presence' for "hi":String (NoMethodError)
Did you mean?  prepend
	from /Users/stanhu/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/irb-1.4.1/exe/irb:11:in `<top (required)>'
	from /Users/stanhu/.asdf/installs/ruby/3.1.2/bin/irb:25:in `load'
	from /Users/stanhu/.asdf/installs/ruby/3.1.2/bin/irb:25:in `<main>'
```

Reply:
You're right, it's the second time I'm trying to use rails helper methods in this repo. Anyway, the sad part is that it doesn't seem like any of the test examples in the whole gem test all that much 😅 it's all stubbed and doesn't reach that code. I'll try to refactor this as part of this PR when I have a minute.

Reply:
Interesting, `activemodel` is a dependency of `openid_connect`, so as a result `activesupport` is available.